ansible/roles/kubernetes-common/defaults/main.yml

```diff
@@ -4,4 +4,5 @@ kubernetes_common_api_ip: 10.10.10.3
 kubernetes_common_primary_interface: enp0s8
 
 # kubelet_extra_args is a dict of arg:value (ie. 'node-ip: 1.1.1.1' for '--node-ip=1.1.1.1')
-kubernetes_common_kubelet_extra_args: {}
+kubernetes_common_kubelet_extra_args:
+  cgroup-driver: systemd
```

Comment:
Not directly related to this change, but I did notice that this is modifying a systemd unit file, which should not be done for recent (at least k8s v1.11+) deployments. Instead these should be passed in through the kubeadm config.